src/peer-connection-utils.spec.ts

```diff
@@ -20,14 +20,16 @@ describe('getLocalDescriptionWithIceCandidates', () => {
     const promise = new PromiseHelper(
       getLocalDescriptionWithIceCandidates(mockPc as unknown as PeerConnection)
     );
-    expect(mockPc.on.mock.calls).toHaveLength(1);
-    expect(mockPc.on.mock.calls[0][0]).toBe(PeerConnection.Events.IceGatheringStateChange);
+    expect(mockPc.Events.iceGatheringStateChange.on.mock.calls).toHaveLength(1);
+    //expect(mockPc.on.mock.calls).toHaveLength(1);
+    //expect(mockPc.on.mock.calls[0][0]).toBe(PeerConnection.Events.IceGatheringStateChange);
     // Grab the listener that was installed
-    const iceGatheringStateListener = mockPc.on.mock.calls[0][1];
+    //const iceGatheringStateListener = mockPc.on.mock.calls[0][1];
+    const iceGatheringStateListener = mockPc.Events.iceGatheringStateChange.on.mock.calls[0][0];
     iceGatheringStateListener({
       target: {
         iceGatheringState: 'gathering',
-      },
+      } as unknown as EventTarget,
```

Comment:
This is actually something I'm surprised ever worked, but I think it's actually kinda better we have to do this.  The argument to this event is `EventTarget` which defines no actual fields: it's basically a form of `any`.  In a real browser, the target here will be the entire `RTCPeerConnection`.  I thought about changing the event to match that, but that might cause problems in the browser and we don't want to have to make a type look like that in the tests, so I did this instead. 

I don't get why this wasn't necessary before, though, as what we passed here definitely _doesn't_ match the definition of `EventTarget`.